clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1479,10 +1479,11 @@ raw2trace_t::read_syscall_template_file()
         syscall_trace_template_encodings_.entry_memref_count(&entry);
         if (entry.type == TRACE_TYPE_MARKER) {
             switch (entry.size) {
-            case TRACE_MARKER_TYPE_SYSCALL:
```

Comment:
Will this break pre-existing template insertion?  Are there existing uses?  Does the version need bumping?

Reply:
It would. I wasn't as concerned about it though, because this feature is still experimental. I noticed op_syscall_template_file doesn't mention it as such though. So I added a note there about the format being prone to change.